### PR TITLE
Added --check_metadata option to enable or disable checking metadata …

### PR DESCRIPTION
**Pull request by mangchiandjjoe**
…service

### CODE REVIEW

File: start_esp/start_esp.py [hunk 1]
```diff
@@ -460,6 +468,10 @@ def make_argparser():
         header, Default value: {xff_trusted_proxy_list}'''.
         format(xff_trusted_proxy_list=DEFAULT_XFF_TRUSTED_PROXY_LIST))
 
+    parser.add_argument('--check_metadata', action='store_true',
```
Comment (qiwzhang) [inline start_esp/start_esp.py:466]: what is the default value?

File: start_esp/start_esp.py [hunk 2]
```diff
@@ -238,6 +238,11 @@ def fetch_service_config(args):
     try:
         # Get the access token
         if args.service_account_key is None:
+            if args.check_metadata == False:
```
Comment (qiwzhang) [inline start_esp/start_esp.py]: this may have problem.   for GKE, we want to fetch access token from metadata server by default

Comment (mangchiandjjoe) [reply start_esp/start_esp.py]: Removed checking args.check_metadata option

File: start_esp/start_esp.py (hunk 1 above)
Comment (mangchiandjjoe) [reply start_esp/start_esp.py:466]: The default value is False. store_true action is handling it.

**Review by qiwzhang — APPROVED** [review]

**Comment by qiwzhang** [comment]
Please e2e test it before merge

**Comment by mangchiandjjoe** [comment]
I will merge the PR first then run pr-e2e test on ESP repo to verify the change.

**Pull request merged**